scripts/Dockerfile.devel

```diff
@@ -1,9 +1,11 @@
-FROM ceph/ceph:v14
+FROM ceph/ceph:v15
 
-ENV GOPATH=/go
+ENV GOPATH=/go \
+	GO111MODULE=on
```

Comment:
Is it this made `on` for `go mod` dependency to work while building ? @Madhu-1

Reply:
Yes humble